mlrun/api/utils/scheduler.py

```diff
@@ -128,15 +129,24 @@ def list_schedules(
         schedules = []
         for db_schedule in db_schedules:
             schedule = self._transform_db_schedule_to_schedule(db_schedule)
+            if include_last_run:
+                schedule = self._enrich_schedule_with_last_run(db_session, schedule)
             schedules.append(schedule)
         return schemas.SchedulesOutput(schedules=schedules)
 
     def get_schedule(
-        self, db_session: Session, project: str, name: str
+        self,
+        db_session: Session,
+        project: str,
+        name: str,
+        include_last_run: bool = False,
     ) -> schemas.ScheduleOutput:
         logger.debug("Getting schedule", project=project, name=name)
         db_schedule = get_db().get_schedule(db_session, project, name)
-        return self._transform_db_schedule_to_schedule(db_schedule)
+        schedule = self._transform_db_schedule_to_schedule(db_schedule)
+        if include_last_run:
+            schedule = self._enrich_schedule_with_last_run(db_session, schedule)
```

Comment:
Put this inside `_transform_db_schedule_to_schedule` and then no need to duplicate no ?